temporalcli/commands.workflow.go

```diff
@@ -442,11 +444,17 @@ func (s *SingleWorkflowOrBatchOptions) workflowExecOrBatch(
 		reason = defaultReason()
 	}
 
+	// Check rps is used together with query
+	if s.Rps != 0 && s.Query == "" {
```

Comment:
This condition will never be true down here (`Query` is always non-empty down here, it's checked above)

Reply:
Good callout 👍 fixed!